core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -25,9 +28,34 @@ import {environment} from "../../../../environments/environment";
 })
 
 
-export class CreateServiceComponent implements OnInit {
+export class CreateServiceComponent implements OnInit , AfterViewInit{
 
   @Output() onClose:EventEmitter<boolean> = new EventEmitter<boolean>();
+  slsFile;
+  deploymentDescriptorText = env_oss.deploymentDescriptorText;
+  typeofservice:boolean=true;
+  typeofplatform:boolean=false;
+  typeofserviceSelected:boolean = false;
+  typeofplatformSelected:boolean = false;
+  typeofruntimeSelected:boolean = false;
+  deploymenttargetSelected:boolean = false;
+  typeOfRuntime:string = "nodejs";
+  serviceTypes = ["api","function","website","custom"];
+  serviceTypesLabels = ["API","Function","Website","Build Custom"]
```

Comment:
Instead of keeping serviceTypes and serviceTypesLabels as arrays, can we map then and use? 
Similarly for -
- platformTypes and platformTypesLabels
- deploymentTargetsList and deploymentTargetsLabels

Reply:
Removed unnecessary code